requirements.txt

```diff
@@ -40,4 +39,7 @@ torchvision
 transformers
 validators
 xxhash==1.3.0
-codecov
\ No newline at end of file
+pandas
```

Comment:
Did you say this worked because we install cython first?
I thought you mentioned that this doesnt work?

Reply:
We need to install pandas, it wasn't added anywhere, so not sure what you mean :)

Reply:
Ah, just caught what you meant. Yes I moved it to the bottom before thinking that the install order mattered, but it apparently doesn't.